src/scala/main/ly/stealth/mesos/kafka/Broker.scala

```diff
@@ -289,12 +301,14 @@ object Broker {
   class Stickiness(_period: Period = new Period("10m")) {
     var period: Period = _period
     @volatile var hostname: String = null
+    @volatile var port: Integer = null
```

Comment:
I would prefer an Option[Int] here rather than a nullable to make it more idiomatic scala.